### PR TITLE
feat(router): React router v6 added to project. Router is wrapping Ap…

### PR DESCRIPTION
**Pull request by VillaCodes**
…p in main.tsx. Recipe read request helper created. Styling has begun on cards. RecipeInfo interface created for typing. RecipePage.tsx is being initialized after this commit.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,10 +22,16 @@ function App() {
   return (
     <FoodprintContextProvider>
       <div className="App">
-        <IngredientsContainer />
-        <Recipes />
-        <header>It is time to scream, my dudes</header>
-        {openPopup && <LandingPagePopup toggle={handleClose}/>}
+        <Routes>
```
Comment (JoelGuizar) [inline src/App.tsx:17]: if we are doing routes wee will need to separate the routes from the component they  are rendering so this page isn't clogged up

File: src/components/Recipes/RecipeItem.css [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+  width:100%;
+}
+
+.body-content {
+  padding-bottom: 40px;
+  font-size: 13px;
+  line-height: 1.8;
+}
+
+.button-primary {
```
Comment (JoelGuizar) [inline src/components/Recipes/RecipeItem.css]: is this bootstrap?

**Review by JoelGuizar — CHANGES_REQUESTED** [review]

**Comment by JoelGuizar** [comment]
also this commit message is a little long -- please ammend

**Review by cguizar25 — DISMISSED** [review]

File: src/utils/SpoonacularRequests.ts [hunk 3]
```diff
@@ -18,6 +26,21 @@ export const fetchData = async(addRecipe: (title:string, id:number, image: strin
   const call = await fetch(`https://api.spoonacular.com/recipes/findByIngredients?apiKey=e0b11b97f0b64f5982330cabe8d0ec9b&ingredients=${ingredientSearch(ingredientList)}`);
 
   const response = await call.json();
+  console.log(response)
 
   onCallHandler(response, addRecipe, 5);
 }
+
+export const readRecipe = async(recipeId: number) => {
+  const url = `https://api.spoonacular.com/recipes/${recipeId.toString()}/information?apiKey=73a82c046a6043fd8fd23732ec9380c7&includeNutrition=false`
```
Comment (JoelGuizar) [inline src/utils/SpoonacularRequests.ts]: lets use JSON stringify instead of toString

Comment (JoelGuizar) [reply src/utils/SpoonacularRequests.ts]: but also I dont think you need either here, sincee its in a template string

File: src/components/Recipes/RecipeItem.css (hunk 2 above)
Comment (VillaCodes) [reply src/components/Recipes/RecipeItem.css]: No bootstrap has been imported or used. I'm changing the naming convention to simply button.

File: src/utils/SpoonacularRequests.ts (hunk 3 above)
Comment (VillaCodes) [reply src/utils/SpoonacularRequests.ts]: Changing this to be string since I will be grabbing the ID from the router params

**Pull request closed without merging**